clink/lib/src/word_classifications.cpp

```diff
@@ -141,7 +141,7 @@ bool word_classifications::equals(const word_classifications& other) const
 
     if (m_face_definitions.size() != other.m_face_definitions.size())
         return false;
-    if (strcmp(m_faces, other.m_faces) != 0)
+    if (strncmp(m_faces, other.m_faces, m_length) != 0)
```

Comment:
Great find, thank you!  It also needs this:
```cpp
if (m_length != other.m_length)
    return false;
```
To save time while also giving credit in github's metrics tracking, I'll merge the PR and then add the check.